### ISSUE TITLE
RFE: layer/descriptor annotations

### ISSUE THREAD

**Issue by vbatts** [openSUSE/umoci#298]
for `umoci insert` a flag similar to `umoci config --manifest.annotation value`, but to attach to the layer descriptor information about that layer

**Comment by cyphar** [comment]
Yeah this is something I've been trying to come up with a UX solution for (in fact we should be able to set annotations for any descriptor in the image), and haven't yet. The most trivial way would be to pass JSON or open an editor with a TOML file, but I'm not sure whether that Kubernetes-like interface has proven to be a good idea (the code in Kubernetes to handle their TOML editor capabilities is worryingly complicated).
